doc-src/ch-docker2sqfs_desc.rst

```diff
@@ -0,0 +1,34 @@
+Synopsis
+========
+
+::
+
+  $ ch-docker2sqfs IMAGE OUTDIR
+
+Description
+===========
+
+Flattens the Docker image tagged :code:`IMAGE` into a Charliecloud squashfs in
+directory :code:`OUTDIR`.
+
```

Comment:
What happens to the resulting tarball? If we remove, we should state it exists only temporarily.